features/unattached_status.feature

```diff
@@ -1,5 +1,32 @@
 Feature: Unattached status
 
+    @series.all
+    @wip
+    Scenario Outline: Unattached status in a ubuntu machine - formatted
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `ua status --format json` as non-root
```

Comment:
Another one: I am testing it only as non-root, because it seems that status with `sudo` will generate the same output

Reply:
I mean, having another call as root will not affect performance that much. But I do agree that this is an overkill